advocacy_docs/community/contributing/styleguide.mdx

```diff
@@ -594,3 +596,34 @@ When specifying dates for human readability, use the DD mmm YYYY format with a s
 
 When specifying dates as solely numbers, use [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) format; YYYY/MM/DD. This is the internationally accepted, disambiguous format and should be used where you may expect the date to be read by automated systems.
 
+## Terminology considerations 
+
+### Ambiguity of the term "cluster"
+
+The term "cluster" can refer to different groups of resources in the technological field. 
+At EDB, the product documentation uses the term "cluster" to refer to not one, but several different objects/concepts.
+To avoid ambiguity, always use an adjective (describing word) to clarify which of the cluster types you are referring to in the specific context.
+
+Here are a few examples:
+
+**Postgres cluster**: Also known as a [database cluster](/epas/latest/fundamentals/epas_fundamentals/terminology/#database-cluster), a Postgres cluster is a Postgres server that hosts a number of databases in a single file system location.
+
+**PGD cluster**: A [PGD cluster](/pgd/latest/terminology/#pgd-cluster) is a [Postgres Distributed](/pgd/latest/) system with a group of multiple redundant database systems in different locations.
+
+**Kubernetes cluster**: A [Kubernetes cluster](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster) is a set of multiple nodes which run containerized applications.
+
+### Postgres distributions
+
+EDB products support three types of Postgres distributions: 
+
+- PostgreSQL (open-source or community Postgres) 
+- EDB Postgres Advanced Server (EPAS) 
+- EDB Postgres Extended Server (PGE) 
+
+Although also referred to as "Postgres flavors," the preferred terms are "Postgres" or "Postgres distributions" to refer to all three options.
+
+## Trademark symbols 
+
+Use the ™ or ® symbol at least upon the first use of the trademark in the text of a page.
```

Comment:
Do we really want to get into this? In Markdown, we can't unfortunately mark first use. Consider other approaches.

Reply:
@djw-m @julienmarcbrown any thoughts? 

Betsy is referring to 
<img width="694" alt="Screenshot 2024-05-29 at 08 49 13" src="https://github.com/EnterpriseDB/docs/assets/98893586/0b801785-e6df-4697-b303-7150b1f15539">

Which links to https://www.enterprisedb.com/trademarks (still not updated with EDB Postgres AI, but Dj is asking about this with the stakeholders)

Reply:
> We should definitely mark trademarks and copyrights for other people's products in first use. But for all of our trademarks? 

With the rebrand, the Postgres (trademarked form) is embedded in our product naming and I believe there is a requirement for that trademark to be acknowledged as in first mention EDB Postgres® AI.

This changes the equation as to whether we can completely omit the ® mark, when it is required contractually. Using it in titles may be problematic for SEO but we will need to see what Algolia and Google make of it.

Reply:
Maybe we can be more specific about this: 

```suggestion
Use the ™ or ® symbol at least upon the first use of the `EDB Postgres® + [product]` trademark in the body text of a page. 

Examples: EDB Postgres® AI, EDB Postgres® Analytics, EDB Postgres® Machine Learning, etc. 

For other trademarks, the documentation site already includes a general disclaimer at the footer of every page called "Trademark" which links to https://www.enterprisedb.com/trademarks.
```